src/main/groovy/com/bmuschko/gradle/docker/tasks/AbstractDockerRemoteApiTask.groovy

```diff
@@ -160,11 +160,10 @@ abstract class AbstractDockerRemoteApiTask extends DefaultTask {
         // Create client
         DockerClient dockerClient = DockerClientBuilder.getInstance(dockerClientConfig).build()
 
-        // register shutdown-hook to close kubernetes client.
-        addShutdownHook {
+        // register buildFinished-hook to close kubernetes client.
```

Comment:
Care to fix this comment and change to `register buildFinished-hook to close docker client`?

Reply:
done

Reply:
Thanks. Always meant to get around to fixing that but then it always slips my mind.